scrapers/koji.py

```diff
@@ -218,9 +218,12 @@ def update_neo4j(self, builds):
             xml_root = ET.fromstring(task_dict['request'])
             commit_hash = None
             for child in xml_root.iter('string'):
+                # Avoids incorrect creation of Commits
+                if '.tar' in child.text:
```

Comment:
This is to vague of a search. What if some metadata is stuffed in there that is unrelated and has ".tar" in it?

Reply:
So I checked a couple of other tasks and they only had the commit hash so I decided to go ahead with this. I even looked through the data from teiid and there doesn't seem to be anything else which can distinguish a tar build. I am open to other ideas :)